### PR TITLE
WT-13405 Bring the Catch2 unit test code in line with the WiredTiger C++ style

### PR DESCRIPTION
**Pull request by luke-pearson**
This change is a style change only. There is a follow up ticket to make the test themselves more consistent which will be more work that what this was.

### CODE REVIEW

File: test/unittest/tests/block/test_block_ckpt.cpp [hunk 1]
```diff
@@ -36,23 +37,23 @@ TEST_CASE("Block helper: __wt_rduppo2", "[block]")
...
     int result = __ut_ckpt_add_blkmod_entry(
-      session->getWtSessionImpl(), blockMods.getWTBlockMods(), offset, len);
+      session->get_wt_session_impl(), block_mods.get_wt_block_mods(), offset, len);
     REQUIRE(result == 0);
 
-    REQUIRE(blockMods.getWTBlockMods()->nbits == expectedBits);
-    REQUIRE(blockMods.getWTBlockMods()->bitstring.memsize == (expectedBits / 8));
-    REQUIRE(blockMods.getWTBlockMods()->bitstring.mem != nullptr);
-    REQUIRE(blockMods.getWTBlockMods()->bitstring.data != nullptr);
+    REQUIRE(block_mods.get_wt_block_mods()->nbits == expectedBits);
```
Comment (mm-ng) [inline test/unittest/tests/block/test_block_ckpt.cpp:53]: Can we change `expectedBits` to the correct case?

File: test/unittest/tests/block/test_bitstring.cpp [hunk 2]
```diff
@@ -6,9 +6,10 @@
  * See the file LICENSE for redistribution information.
  */
 
-#include "wt_internal.h"
 #include <catch2/catch.hpp>
 
+#include "wt_internal.h"
+
 TEST_CASE("Bitstring macros: __bit_byte", "[bitstring]")
```
Comment (mm-ng) [inline test/unittest/tests/block/test_bitstring.cpp:13]: Can you please fix the `bitVectorSize` and `bitVector` variables too?

**Review by jiechenbo — APPROVED** [review]
LGTM! after Monica's changes.

File: test/unittest/tests/block/test_bitstring.cpp (hunk 2 above)
Comment (luke-pearson) [reply test/unittest/tests/block/test_bitstring.cpp:13]: ac72d1a85

File: test/unittest/tests/block/test_block_ckpt.cpp (hunk 1 above)
Comment (luke-pearson) [reply test/unittest/tests/block/test_block_ckpt.cpp]: ac72d1a85

**Comment by luke-pearson** [comment]
I've made the suggested changes from Monica and resolved merge conflicts due to Jie's recent PR going in. Could we merge this before we merge anything else to minimize further merge conflicts?

**Review by mm-ng — APPROVED** [review]
LGTM

**Pull request merged**